sepfinder.py

```diff
@@ -93,11 +93,27 @@ def on_text(update, ctx):
             return update.message.reply_text('Invalid input.')
 
         r = session.get(f'https://api.ipsw.me/v4/device/{device["identifier"]}')
-
-        if not r.ok:
-            return update.message.reply_text('Unable to communicate with ipsw.me API, please try again later.')
-
-        device = r.json()
+        rb = session.get(f'https://api.m1sta.xyz/betas/{device["identifier"]}')
+        if r.ok:
+            device = r.json()
+            ctx.bot_data['ipswme_failed'] = False
+        else:
+            device = {
+                'name': device['name'],
+                'identifier': device['identifier'],
+                'boards': device['boards'],
+                'firmwares': [],
+            }
+            update.message.reply_text('Unable to communicate with ipsw.me API.')
+            ctx.bot_data['ipswme_failed'] = True
```

Comment:
```suggestion
            ctx.user_data['ipswme_failed'] = True
```
It shouldn't use `bot_data`, could cause issues if multiple users are using the bot at once.

Reply:
Hmm, technically I use it for the devices dict but it never actually reuses the old data for new sessions anyway so I should probably change that too.